homeassistant/components/wemo/switch.py

```diff
@@ -116,7 +116,8 @@ def detail_state(self) -> str:
             if standby_state == StandbyState.STANDBY:
                 return STATE_STANDBY
             return STATE_UNKNOWN
-        assert False  # Unreachable code statement.
+        # Unreachable code statement.
+        raise RuntimeError
```

Comment:
Can we remove `pragma no cover` here?
```suggestion
    raise AssertionError("Should be unreachable")
```